### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by ryt2n**
The links on the HP do not work on my branch.  Easy to correct through.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -11,3 +11,5 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+@import "components/buttons";
```
Comment (contatc2) [inline app/assets/stylesheets/application.scss]: should be imported in components/index instead

File: app/assets/stylesheets/components/_index.scss [hunk 2]
```diff
@@ -4,3 +4,6 @@
 @import "navbar";
 @import "cards";
 
+.container{
```
Comment (contatc2) [inline app/assets/stylesheets/components/_index.scss]: do you need to do that if you delete the container class from the application.html.erb ?

Comment (contatc2) [reply app/assets/stylesheets/components/_index.scss]: i see you use it somewhere else

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -1,2 +1,18 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+  <%= image_tag "HomeHero.jpg", width: '100%' %>
+ <div class="container">
+  <h2>Find a real person</h2>
+  IRL facilitates matchmaking by
+  leaverging your friends, ensuring genuine interactions and removing the hassle in introductions.  You can nominate match-makers and enable them to alert you when they find a potentional suitable match for you, while always leaving you in total control.
```
Comment (contatc2) [inline app/views/pages/home.html.erb]: leveraging

**Comment by ryt2n** [comment]
Let me know what you think?

**Pull request merged**